vm/src/builtins/weakproxy.rs

```diff
@@ -136,6 +141,25 @@ impl SetAttr for PyWeakProxy {
     }
 }
 
+impl Comparable for PyWeakProxy {
+    fn cmp(
+        zelf: &crate::Py<Self>,
+        other: &PyObject,
+        op: PyComparisonOp,
+        vm: &VirtualMachine,
+    ) -> PyResult<PyComparisonValue> {
+        println!("richcompare");
+        op.eq_only(|| {
```

Comment:
Does weakproxy only support eq operation?
What happens if the proxy object is `int` and used for `>` operation?

Reply:
I checked cpython source code and it simply calls `PyObject_RichCompare`, which is `PyObjectRef::rich_compare` in RustPython.

Reply:
I checked cpython source code and it simply calls `PyObject_RichCompare`, which is `PyObjectRef::rich_compare` in RustPython.